packages/web-pkg/src/components/FilesList/ResourceLink.vue

```diff
@@ -8,7 +8,7 @@
     class="oc-resource-link"
     no-hover
     @dragstart.prevent.stop
-    @click="emitClick"
+    @click="(e: MouseEvent) => emitClick(e)"
```

Comment:
It should work the way it was before.

```suggestion
    @click="emitClick"
```